src/reference/asciidoc/amqp.adoc

```diff
@@ -1705,6 +1705,12 @@ new MessageListenerAdapter((ReplyingMessageListener<String, String>) data -> {
 ----
 ====
 
+Starting with version 2.2, we deprecate method `buildListenerArguments(Object)` and provide
```

Comment:
I prefer the formal language in the docs without referring to any person.
So, not `we`, but "the `MessageListenerAdapter.buildListenerArguments(Object)`"  has been deprecated.

Also , please, follow the rule *One sentence per like*: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Your next sentence after period starts with lower case, for example.